### DISCUSSION TITLE
soapysdr missing modules formulae

### DISCUSSION THREAD

**Discussion by maichai** [Homebrew/discussions#1756] [Everyday usage]
soapysdr has a number of additional modules and even provides brew formulae for that, at least it seems so, but the only `module brew search soapy` finds is `soapyrtlsdr`. Missing modules:

```
brew install soapyremote
brew install soapyaudio
brew install soapyhackrf
brew install soapybladerf
brew install soapyairspy
brew install soapyairspyhf
brew install soapyosmo
brew install soapyuhd
brew install soapyredpitaya
brew install soapyiris
brew install limesuite
```

Most sorely missed: `soapyhackrf` and `soapyosmo`

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by carlocab** [comment]
If these meet the [requirements](https://docs.brew.sh/Acceptable-Formulae) you're welcome to make a pull request: https://docs.brew.sh/Adding-Software-to-Homebrew
